### PR TITLE
pxview: init at 0.2.5

### PR DESCRIPTION
**Pull request by winpat**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by mkaito** [comment]
@GrahamcOfBorg build pxview

**Review by mkaito — APPROVED** [review]

File: pkgs/development/libraries/pxlib/default.nix [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  src = fetchurl {
+    url = "mirror://sourceforge/${pname}/${pname}-${version}.tar.gz";
+    sha256 = "1yafwz4z5h30hqvk51wpgbjlmq9f2z2znvfim87ydrfrqfjmi6sz";
+  };
+
+  buildInputs = [ intltool ];
+
+  meta = with stdenv.lib; {
+    description = "Library to read and write Paradox files";
+    homepage = http://pxlib.sourceforge.net/;
```
Comment (veprbl) [inline pkgs/development/libraries/pxlib/default.nix:15]:
```suggestion
    homepage = "http://pxlib.sourceforge.net/";
```

File: pkgs/development/tools/pxview/default.nix [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+  configureFlags = [ "--with-pxlib=${pxlib.out}" ];
+
+  # https://sourceforge.net/p/pxlib/bugs/12/
+  NIX_LDFLAGS = "-lm";
+  NIX_CFLAGS_COMPILE = "-Wno-format-security";
+
+  meta = with stdenv.lib; {
+    description = "Program to convert Paradox databases";
+    homepage = http://pxlib.sourceforge.net/pxview/;
```
Comment (veprbl) [inline pkgs/development/tools/pxview/default.nix:22]:
```suggestion
    homepage = "http://pxlib.sourceforge.net/pxview/";
```

File: pkgs/development/libraries/pxlib/default.nix [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+stdenv.mkDerivation rec {
+  pname = "pxlib";
+  version = "0.6.8";
+  src = fetchurl {
+    url = "mirror://sourceforge/${pname}/${pname}-${version}.tar.gz";
+    sha256 = "1yafwz4z5h30hqvk51wpgbjlmq9f2z2znvfim87ydrfrqfjmi6sz";
+  };
+
+  buildInputs = [ intltool ];
```
Comment (veprbl) [inline pkgs/development/libraries/pxlib/default.nix:11]:
```suggestion
  nativeBuildInputs = [ intltool ];
```

File: pkgs/development/tools/pxview/default.nix [hunk 4]
```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchurl , pkgconfig, perl, perlPackages, pxlib }:
```
Comment (veprbl) [inline pkgs/development/tools/pxview/default.nix:1]:
```suggestion
{ stdenv, fetchurl, pkgconfig, perl, perlPackages, pxlib }:
```

File: pkgs/development/tools/pxview/default.nix [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+    sha256 = "1kpdqs6lvnyj02v9fbz1s427yqhgrxp7zw63rzfgiwd4iqp75139";
+  };
+
+  buildInputs = [ pxlib perl ] ++ (with perlPackages; [ libxml_perl ]);
+  nativeBuildInputs = [ pkgconfig ];
+
+  configureFlags = [ "--with-pxlib=${pxlib.out}" ];
+
+  # https://sourceforge.net/p/pxlib/bugs/12/
+  NIX_LDFLAGS = "-lm";
```
Comment (veprbl) [inline pkgs/development/tools/pxview/default.nix:17]:
```suggestion
  LDFLAGS = "-lm";
```
As suggested by #79303

File: pkgs/development/tools/pxview/default.nix [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+  NIX_CFLAGS_COMPILE = "-Wno-format-security";
```
Comment (veprbl) [inline pkgs/development/tools/pxview/default.nix:18]:
```suggestion
  hardeningDisable = [ "format" ];
```

**Comment by winpat** [comment]
@veprbl Thank you for your feedback. I should be able to incorporate it in the following weekend.

**Review by veprbl — APPROVED** [review]

**Pull request merged**